utils/summary.jsx

```diff
@@ -47,19 +47,22 @@ export const formatData = (componentProps, formData) => {
     return formatAddress(formData[name], name, label);
   }
   if (component === 'Checkbox' && options) {
+    const stepOptions =
+      typeof options === 'function' ? options(formData) : options;
+
     return {
       key: name,
       title: label,
       value:
-        typeof options[0] === 'string'
+        typeof stepOptions[0] === 'string'
           ? formData[name]
-          : formData[name]?.map((data) =>
-              multiValue(
+          : formData[name]?.map((data) => {
```

Comment:
![Screenshot from 2021-07-08 09-59-49](https://user-images.githubusercontent.com/29123700/124894195-5873a700-dfd3-11eb-99d0-9beeee0aa1ec.png)

Looks like we're trying to call .map() against a string and it throws an error when trying to create the relationship.

Reply:
Thanks for the spot! I've been trying to debug this and it turns out that either the `Checkbox` or `FormWizard` is returning multiple types: string, an array of strings, empty array or boolean. 

It seems like we'll have to handle that as I'm struggling to figure out which one it is and am wary of changing things in `FormWizard` especially. Let me know if you have any suggestions or spare time today, otherwise I will just have to handle the different types.

Reply:
i have no insight to offer other than this is an excellent reason to use the new form builder instead!

Reply:
@JohnFarrellDev, I've pushed up the fix for this - not ideal but I think it's fine for now. 

@jhackett1, could you 🍐 with me on how we'd do that tomorrow? I'm going to merge this PR though as the functionality is there and I've branched off of this branch with some other functionality but I'd definitely be up with  seeing how the new form builder works!